plugin/ToolBar.qml

```diff
@@ -35,6 +35,7 @@ PanelBackground {
             autoHideTimer.start()
         } else {
             autoHideTimer.stop()
+            _active = true
```

Comment:
You're right, something like ```_active = (offset > 0)``` would allow to change autoShowHide property without changing the visibility of the toolbar: a hidden one stays hidden, and a transiting one (either showing or masking) will stay visible.